backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/utils/Modifier.kt

```diff
@@ -58,23 +58,23 @@ internal inline fun Modifier.applyIf(predicate: Boolean, block: Modifier.() -> M
     return if (predicate) block() else this
 }
 
-// todo: in Compose 1.6, there's no way to provide an indication using the Node-like API.
-// It will be added in Compose 1.7, so we'll migrate it after that
-@Suppress("ModifierComposed")
-internal fun Modifier.clickable(
+@Suppress("ModifierComposable")
```

Comment:
the main compose docs now mention it's better to use a composable function than the composed API. it's not really feasible to move to the node api for this as the whole clickable node is internal